tests/bits/parameter_handler_backslash_06.expect=diff.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::a, b, c
+DEAL::a, b, c
```

Comment:
Is it a mistake that this file has no newline at the end of its last line? I think the test actually outputs a newline...